src/molecules/lists/radio-list.test.tsx

```diff
@@ -16,4 +21,32 @@ describe("RadioList", () => {
         // Assert
         expect(getByText(expected)).not.toBeNull();
     });
+
+    test("when items prop is empty, radio-list returns", () => {
```

Comment:
Since `result` is null, shouldn't the name be something different than `radio-list returns`?